src/styles/blocks/calendar/calendar-block.scss

```diff
@@ -0,0 +1,42 @@
+.calendar {
+  max-width: 710px;
+  display: flex;
+  flex-wrap: wrap;
+  gap: 1px;
+
+  @each $day, $number in $days {
+    &--start--#{$day} :nth-child(1) {
+      margin-left: ($WidthOf-dayBlock + 1) * $number;
+    }
+  }
+
+  @for $index from 28 through 31 {
+    &--month-length--#{$index} :nth-child(n + #{$index + 1}) {
+      visibility: hidden;

```

Comment:
Display none will better her because visibility only hide block but it remains in the stream

```suggestion
      display: none;
```

![image](https://user-images.githubusercontent.com/63306833/125913894-268e7b4d-fb47-4be6-adf8-d1595e57ae52.png)